src/wallet/ismine.h

```diff
@@ -17,13 +17,13 @@ class CScript;
 /** IsMine() return codes */
 enum isminetype : unsigned int
 {
-    ISMINE_NO         = 0,
-    ISMINE_WATCH_ONLY = 1 << 0,
-    ISMINE_SPENDABLE  = 1 << 1,
-    ISMINE_USED       = 1 << 2,
-    ISMINE_ALL        = ISMINE_WATCH_ONLY | ISMINE_SPENDABLE,
-    ISMINE_ALL_USED   = ISMINE_ALL | ISMINE_USED,
-    ISMINE_ENUM_ELEMENTS,
+    ISMINE_NO         = 0, //!< the scriptPubKey is not contained in the wallet
```

Comment:
nit, could keep comments aligned, like https://github.com/bitcoin/bitcoin/blob/463eab5e1418a592036e7bf9bf46f66fe6462435/src/chain.h#L40-L51

Reply:
> nit, could keep comments aligned, like
> 
> https://github.com/bitcoin/bitcoin/blob/463eab5e1418a592036e7bf9bf46f66fe6462435/src/chain.h#L40-L51

done.  thx for pointing it out